### PR TITLE
feat: master key improvements

### PR DESCRIPTION
**Pull request by reneaaron**
### Describe the changes you have made in this PR

 - 2 step process (explanation, then generate)
 - general layout improvements (especially for smaller screen heights)
 - new colors / border styles applied to account details / settings

### CODE REVIEW

File: src/app/components/AccountDetailLayout/index.tsx [hunk 1]
```diff
@@ -50,12 +48,11 @@ function AccountDetailLayout() {
             >
               {account.name}
             </h2>
+            <span>/</span>
+            <span>{t("title1")}</span>
           </div>
-          <div className="flex-row justify-center flex items-center text-gray-500 text-sm dark:text-neutral-500">
-            {account.connector}
```
Comment (rolznz) [inline src/app/components/AccountDetailLayout/index.tsx:55]: is the only place to see the connector type on the accounts list page now?

**Review by rolznz — APPROVED** [review]
tACK

File: src/app/screens/Accounts/GenerateMnemonic/new.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    navigate(`/accounts/${id}`);
+  }
+
+  return (
+    <Container>
+      <ContentBox>
+        <h1 className="font-bold text-2xl dark:text-white">{t("title")}</h1>
+        <img
+          src={`assets/images/master_key_${theme}.png`}
+          alt="Pin your Alby extension"
```
Comment (reneaaron) [inline src/app/screens/Accounts/GenerateMnemonic/new.tsx:30]:
```suggestion
          alt="Master Key"
```

File: src/app/components/AccountDetailLayout/index.tsx (hunk 1 above)
Comment (reneaaron) [reply src/app/components/AccountDetailLayout/index.tsx:55]: Yes, that doesn't fit into the header bar anymore. :man_shrugging:

**Pull request merged**